packages/@dcl/inspector/src/components/EntityInspector/EntityInspector.tsx

```diff
@@ -1,18 +1,31 @@
+import { Entity } from '@dcl/ecs'
+
 import { useSelectedEntity } from '../../hooks/sdk/useSelectedEntity'
+import { withSdk } from '../../hoc/withSdk'
 
 import { SceneInspector } from './SceneInspector'
 import { TransformInspector } from './TransformInspector'
 import { GltfInspector } from './GltfInspector'
 import './EntityInspector.css'
 
-export const EntityInspector: React.FC = () => {
+export const EntityInspector = withSdk(({ sdk }) => {
   const entity = useSelectedEntity()
 
-  const inspectors = [SceneInspector, TransformInspector, GltfInspector]
+  if (entity !== null) {
+    const nameComponent = sdk.components.Name.getOrNull(entity)
+    const label = entity === 0 as Entity ? 'Scene' : (nameComponent ? nameComponent.value : entity.toString())
+
+    const inspectors = [SceneInspector, TransformInspector, GltfInspector]
+
+    return (
+      <div className="EntityInspector" key={entity}>
+        <div className="entity-label">{label}</div>
+        {inspectors.map((Inspector, index) => entity !== null && <Inspector key={index} entity={entity} />)}
+      </div>
+    )
+  } else {
+    return <div className="EntityInspector"></div>
+  }
+})
```

Comment:
maybe we can avoid returning two different jsx by

```ts
const getLabel = (sdk: SdkContextValue, entity: Entity) => {
  const nameComponent = sdk.components.Name.getOrNull(entity)
  return entity === 0 as Entity ? 'Scene' : (nameComponent ? nameComponent.value : entity.toString())
}

export const EntityInspector = withSdk(({ sdk }) => {
  const entity = useSelectedEntity()
  const inspectors = [SceneInspector, TransformInspector, GltfInspector]

  const label = entity ? getLabel(sdk, entity) : null

  return (
    <div className="EntityInspector" key={entity}>
      <div className="entity-label">{label}</div>
      {inspectors.map((Inspector, index) => entity !== null && <Inspector key={index} entity={entity} />)}
    </div>
  )
})
```

what do you think?

Reply:
Sounds good.